features/kernel/src/main/feature/feature.xml

```diff
@@ -114,13 +61,11 @@
     </feature>
 
     <!-- Override the pax-web pax-http-whiteboard feature and remove the vulnerable pax-web-jsp bundle-->
-    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="6.0.6_1">
+    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="${pax.web.version}_1">
```

Comment:
:+1: I reviewed this feature definiton with the original: https://github.com/ops4j/org.ops4j.pax.web/blob/web-6.0.11/pax-web-features/src/main/resources/features.xml#L124-L136/.